### PR TITLE
added vlipboard

### PR DESCRIPTION
**Pull request by asvvvad**
My first v module that adds basic support for Wayland and support for Termux aswell to the V clipboard module

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -133,6 +133,7 @@
 ### Operating System
 
 - [clipboard](https://github.com/vlang/v/tree/master/vlib/clipboard) - V module for interacting with the OS clipboard. Fully cross-platform.
+- [vlipboard](https://github.com/asvvvad/vlipboard) - An easy to use wrapper of clipboard with Wayland and Termux support
```
Comment (alexesprit) [inline README.md:136]: Please add a dot symbol at the end of the sentence to pass tests.

Comment (asvvvad) [reply README.md]: Fixed. sorry about that it's 9am and still haven't slept good night!

**Comment by alexesprit** [comment]
Thanks! It's still failing, but it's another issue.

**Comment by asvvvad** [comment]
Yes I noticed, I looked into the logs and it seems a link that's behind cloudflare and isn't actually down^^

**Comment by medvednikov** [comment]
Cool! Why not add it to the main `clipboard` module?

**Comment by thecodrr** [comment]
@medvednikov it uses `wl-clipboard` underneath as a dependency i.e. it is just a wrapper around `wl-copy`.

**Comment by asvvvad** [comment]
@medvendnikov Thank you! like @thecodrr it depend on external programs since Wayland is tricky and there is no way to do it on Termux without that as well. It's how a [clipboard module for Go](https://github.com/atotto/clipboard) does it as I mentioned in https://github.com/vlang/v/issues/4859#issue-616176197

**Pull request merged**